apps/api/tests/api/application-api.test.ts

```diff
@@ -125,4 +125,47 @@ describe('Application API', () => {
 
 		after(() => {});
 	});
+
+	describe('Get Application by ID', () => {
+		it('should successfully be able to find an application with an ID', async () => {
+			const applicationRecordsResult = await applicationService.listApplications({ user_id });
+
+			assert.ok(applicationRecordsResult.success);
+			assert.ok(Array.isArray(applicationRecordsResult.data) && applicationRecordsResult.data[0]);
```

Comment:
So in the actions/revisions service branch I'm adding `export const testApplicationId = 1;` to `testUtils` and then using it for test cases like this where you need to GET an ID

Because every test spins up it's own container and then runs `addInitialApplications` we can count on testApplication with id 1 to exist every time

Optional, but you can replace lines 131-136 and testApplicationId / `import { testApplicationId } from testUtils` if you wanted